swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -298,7 +264,7 @@ Let's test the use case by writing a test case that sends a request to the servi
     final http:Client cl = check new (string `http://localhost:${port}/healthcare/doctors`);
```

Comment:
"Step 3: Build and run the service" terminal has clear?

Reply:
Yeah. it automatically run a `clear`